src/main/java/com/gmail/nossr50/skills/mining/MiningManager.java

```diff
@@ -175,66 +210,56 @@ public void remoteDetonation() {
      * @param event The {@link EntityExplodeEvent}
      */
     public void blastMiningDropProcessing(float yield, EntityExplodeEvent event) {
-        if (yield == 0)
-            return;
+        if (yield == 0) return;
 
         var increasedYieldFromBonuses = yield + (yield * getOreBonus());
-        // Strip out only stuff that gives mining XP
```

Comment:
Deleting comments is an unusual thing to do in a PR, try to keep your diff minimal